manifest.yml

```diff
@@ -0,0 +1,7 @@
+---
+applications:
+- name: cfpyapi
+  command: python app.py
+  type: application
+  services:
+  - hurricane
```

Comment:
Is this the database service name?

Reply:
yes, if someone replicates this that'll bind their app to the their db upon creation instead of having to manually do it in the cli

Reply:
The naming is a bit confusing. Something like `example-website-api-database` is easier to understand.